Jellyfin.Server.Implementations/Jellyfin.Server.Implementations.csproj

```diff
@@ -31,6 +31,14 @@
     <PackageReference Include="Microsoft.EntityFrameworkCore.Relational" />
   </ItemGroup>
 
+  <ItemGroup>
+    <PackageReference Include="CacheManager.Core" />
+    <PackageReference Include="CacheManager.Microsoft.Extensions.Caching.Memory" />
+    <PackageReference Include="CacheManager.Serialization.Bond" />
+    <PackageReference Include="EFCoreSecondLevelCacheInterceptor.CacheManager.Core" />
+    <PackageReference Include="Newtonsoft.Json" />
```

Comment:
Why does this add Newtonsoft.Json? Shouldn't that show up as a transient package? I don't see any new using statements using it specifically.